flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/python/PythonTableAggregateFunction.java

```diff
@@ -120,6 +163,15 @@ public TypeInference getTypeInference(DataTypeFactory typeFactory) {
         if (inputTypes != null) {
             builder.typedArguments(inputTypes);
         }
+
```

Comment:
`line 152: TypeConversions.fromDataTypeToLegacyInfo(resultType)`

Does it still work?